admin/app/components/solidus_admin/ui/forms/address/component.rb

```diff
@@ -1,15 +1,83 @@
 # frozen_string_literal: true
 
 class SolidusAdmin::UI::Forms::Address::Component < SolidusAdmin::BaseComponent
-  def initialize(addressable:, form_field_name:, disabled: false, include_name_field: true)
+  DEFAULT_FIELDS = %i[
```

Comment:
I think if we make these two constants private methods one could inherit from this component and override this in order to customize its one address form more easily.

Reply:
I'd love for us to favour composition-based extension approaches rather than inheritance-based ones. I know that SolidusAdmin is pre-1.0, but I've already seen the pain of changes to components breaking my subclasses. I think in this case we should _not_ provide the subclass option and just let people wrap the class if they want.